frontend/src/pages/TermPlanner/UnplannedColumn/index.less

```diff
@@ -1,18 +1,75 @@
-.unplannedBox {
-  grid-row-start: 2;
+.unplannedContainer {
+  grid-row-start: 1;
   grid-row-end: span 10;
   grid-column-start: 5;
+  display: flex;
+  flex-direction: column;
+  place-self: stretch;
+  min-width: 15em;
+  align-items: center;
+}
+
+.summerUnplannedContainer {
+  grid-column-start: 6;
+}
+
+.unplannedTitle {
+  position: absolute;
+  font-weight: bold;
+  top: calc(
+    var(--navbar-height) + 
+    var(--option-header-height) + 
+    var(--tp-main-container-padding) + 
+    var(--tp-planner-container-margin) 
+  );
+}
+
+.unplannedBox {
+  position: absolute;
+  overflow-y: auto;
+  top: calc(
+    var(--navbar-height) + 
+    var(--option-header-height) + 
+    var(--tp-main-container-padding) + 
+    var(--tp-planner-container-margin) + 
+    var(--tp-term-box-margin) +
+    var(--tp-grid-item-font-size)
+  );
+  height: calc(
+    100vh - (
+      var(--navbar-height) + 
+      var(--option-header-height) + 
+      var(--tp-main-container-padding) + 
+      3 * var(--tp-planner-container-margin) + 
+      2 * var(--tp-term-box-margin) + 
+      var(--tp-grid-item-font-size)
+    )
+  );
+  min-width: 20em;
+  max-width: 20em;
   margin: 1em;
-  border: 1px solid #D9D9D9;
+  border: 2px solid #9254de;
```

Comment:
I like the border-color, but imo the px should be between 1.5px to 2px? 2px feels a bit to thick for the border. But totally up to you on which looks best. Don't mind either way :D